packages/bigtable/src/index.js

```diff
@@ -309,24 +317,19 @@ function Bigtable(options) {
     return new Bigtable(options);
   }
 
-  var baseUrl = 'bigtable.googleapis.com';
+  var defaultApiEndpoint = 'bigtable.googleapis.com';
   var adminBaseUrl = 'bigtableadmin.googleapis.com';
 
-  var customEndpoint = options.apiEndpoint ||
-    process.env.BIGTABLE_EMULATOR_HOST;
-
-  if (customEndpoint) {
-    baseUrl = customEndpoint;
-    adminBaseUrl = baseUrl;
-  }
-
   var config = {
-    baseUrl: baseUrl,
-    customEndpoint: !!customEndpoint,
+    environmentVariables: [
+      'GOOGLE_CLOUD_BIGTABLE_ENDPOINT',
+      'BIGTABLE_EMULATOR_HOST'
+    ],
+    defaultApiEndpoint: 'bigtable.googleapis.com',
     protosDir: path.resolve(__dirname, '../protos'),
     protoServices: {
       Bigtable: {
-        baseUrl: baseUrl,
+        baseUrl: defaultApiEndpoint,
```

Comment:
Do we still need this? I thought `Service` would default this to the proper API endpoint automatically.